pipenv/core.py

```diff
@@ -2290,7 +2287,9 @@ def do_run_posix(script, command):
                 err=True,
             )
         sys.exit(1)
-    os.execl(command_path, command_path, *script.args)
+    os.execl(
+        command_path, command_path, *[os.path.expandvars(arg) for arg in script.args]
```

Comment:
At this point I think `os.execv` (which takes a sequence directly) is better than `execl`.